### PR TITLE
GG-25459 Start agent only if user successfully logged in.

### PR DESCRIPTION
**Pull request by akuznetsov-gridgain**

### CODE REVIEW

File: modules/web-console/e2e/testcafe/fixtures/auth/login.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {dropTestDB, insertTestUser, resolveUrl} from '../../environment/envtools';
+import {pageSignin} from '../../page-models/pageSignin';
+import {userMenu} from '../../components/userMenu';
+import {WebSocketHook} from "../../mocks/WebSocketHook";
+
+fixture.only('Login')
```
Comment (Klaster1) [inline modules/web-console/e2e/testcafe/fixtures/auth/login.js]:
```suggestion
fixture('Login')
```

Comment (akuznetsov-gridgain) [reply modules/web-console/e2e/testcafe/fixtures/auth/login.js]: Fixed

File: modules/web-console/e2e/testcafe/fixtures/auth/login.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        await t.addRequestHooks(t.ctx.ws = new WebSocketHook().use(() => t.ctx.touched = true));
+    })
+    .after(async() => {
+        await dropTestDB();
+    })
+    .afterEach(async(t) => {
+        t.ctx.ws.destroy();
+    });
+
+test('Ivalid email', async(t) => {
```
Comment (akuznetsov-gridgain) [inline modules/web-console/e2e/testcafe/fixtures/auth/login.js]:
```suggestion
test('Invalid email', async(t) => {
```

File: modules/web-console/e2e/testcafe/fixtures/auth/login.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (akuznetsov-gridgain) [reply modules/web-console/e2e/testcafe/fixtures/auth/login.js]: Fixed

**Pull request merged**